app.py

```diff
@@ -11,17 +11,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import argparse
 import logging
 from os import mkdir
-from os.path import exists, isdir
+from os.path import isdir
 from pathlib import Path
 
 import streamlit as st
 
 from data_measurements import dataset_statistics, dataset_utils
 from data_measurements import streamlit_utils as st_utils
 
+"""
+Examples:
+# When not in deployment mode
+streamlit run app.py -- --live=False
```

Comment:
is there an extra `--` here by mistake?

Reply:
Also, with argparse, you can do this to store booleans if you dont want to add the `=False` in the command line. But either way seems ok to me!

```
parser.add_argument('--foobar', action='store_true')
parser.add_argument('--foonley', action='store_false')
```

Reply:
Nope!

And thanks! I always use store_true, forgot about store_false !!!